client/src/components/landing/components/LandingTop.js

```diff
@@ -12,6 +12,7 @@ import {
   TencentLogo,
   AlibabaLogo
 } from '../../../assets/images/components';
+import MobileAd from '../../MobileAd/mobile-ad-modal';
```

Comment:
Since the other file has been commented out then this import and the component reference in this file will give errors

Reply:
That's correct, at the time I needed to wait because merging main in to the branch wouldn't work.

Reply:
There was some error with prettier and @ojeytonwilliams has fixed and merged his pull request